torch_xla/csrc/tensor.cpp

```diff
@@ -466,24 +466,21 @@ void XLATensor::addcmul_(const at::Scalar& value, const XLATensor& tensor1,
   SetIrNode(GetIrNode() + mul * constant);
 }
 
+xla::int64 XLATensor::size(int dim) const {
+  const xla::Shape& xla_shape = this->shape().get();
```

Comment:
Why this-> ?

Reply:
I was using `shape` to the left hand side (with auto), then I renamed and forgot to clean up the right hand side. Done.